### ISSUE TITLE
"Silently" catching entity tracking error

### ISSUE THREAD

**Issue by jermangarc** [CrazyPants/EnderIO#1780]
Not sure if this is a Mekanism bug or EnderIO or something completely different. But I'm getting server console spam of the log below over and over again, causing server lag spikes every 10-15 sec's. Note: Server and client does NOT crash, but game is very laggy because of the spamming. Any input is appreciated. Thanks. 

http://pastebin.com/PazfKitU

EnderIO-1.7.10-2.2.7.325
Mekanism-1.7.10-7.1.2.157

**Comment by MatthiasMann** [comment]
Seems like you placed the wireless charger on a chunk boundary and the other chunk is not loaded - and checking the redstone status seems to load that chunk.

**Comment by jermangarc** [comment]
Great! Thanks for quick response. I'll check my users' setup. Thanks for your input. I'll post my findings.

**Comment by jermangarc** [comment]
Great! Thanks for quick response. I'll check my users' setup. Thanks for your input. I'll post my findings.

**Comment by jermangarc** [comment]
Yup, chunk boundary issue. One of my user's was using chickenchunks but I'm not sure where his charger is at, so I increased his range to solve the problem. Thanks again.

**Comment by Fogger** [comment]
I also have this issue in the Nether without any chunk loaders.

I found that it only occurs when both EnderIO and ExU are loaded.  I can remove every mod, leaving just these 2 and it will still spam/lag, but as soon as I remove either EnderIO or ExU, the spam/lag is gone.

I also discovered that placing a skeleton spawner in the Nether turns into a Wither Skeleton spawner, but spawns nothing & almost freezes any client that is near. (<1 fps)
